src/ocaml_lsp.ml

```diff
@@ -63,17 +59,10 @@ module Experimental_capabilities = struct
       |> Option.value ~default:false
     in
     try
-      let interfaceSpecificLangId = has_capability "interfaceSpecificLangId" in
       let handleSwitchImplIntf = has_capability "handleSwitchImplIntf" in
-      let handleInferIntf = has_capability "handleInferIntf" in
       let handleTypedHoles = has_capability "handleTypedHoles" in
       let handleTypeEnclosing = has_capability "handleTypeEnclosing" in
-      { interfaceSpecificLangId
-      ; handleSwitchImplIntf
-      ; handleInferIntf
-      ; handleTypedHoles
-      ; handleTypeEnclosing
-      }
+      { handleSwitchImplIntf; handleTypedHoles; handleTypeEnclosing }
```

Comment:
Actually, these probably means errors somewhere else since they should be used to check a feature availability before using it.

And there is indeed a copy-pasting error line 241:
```ocaml
let can_handle_infer_intf t = t.experimental_capabilities.handleSwitchImplIntf
```

It should be `t.experimental_capabilities.handleInferIntf`.

`interfaceSpecificLangId` seems to be a remnant of the past however, it looks like it was used at some point to check the lsp version ?

Reply:
It certainly seems to have been introduced for that purpose. But I think it's no longer necessary, so I think it can be removed from the LSP side as well.

- https://github.com/ocaml/ocaml-lsp/pull/189
- https://github.com/ocamllabs/vscode-ocaml-platform/pull/209